packages/yum/groonga-release.spec

```diff
@@ -24,9 +24,9 @@ Groonga release files
 
 %{__install} -Dp -m0644 RPM-GPG-KEY-groonga %{buildroot}%{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-groonga
 %{__install} -Dp -m0644 RPM-GPG-KEY-groonga-RSA4096 %{buildroot}%{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-groonga-RSA4096
-
 %{__install} -Dp -m0644 groonga.repo %{buildroot}%{_sysconfdir}/yum.repos.d/groonga.repo
```

Comment:
Can we remove this?

Reply:
If we remove this line, `make build-release-rpm` occurs build error as below.

```
RPM build errors:
    Directory not found: /home/horimoto/rpm/BUILDROOT/groonga-release-1.5.0-1.x86_64/etc/yum.repos.d
    File not found: /home/horimoto/rpm/BUILDROOT/groonga-release-1.5.0-1.x86_64/etc/yum.repos.d/groonga.repo
Failed rpmbuild -ba /home/horimoto/rpm/SPECS/groonga-release.spec
```

Therefore,  If we remove it, we need addition `%{__mkdir_p} %{buildroot}%{_sysconfdir}/yum.repos.d/` instead of `%{__install} -Dp -m0644 groonga.repo %{buildroot}%{_sysconfdir}/yum.repos.d/groonga.repo`.

Reply:
`%{__install} -d %{buildroot}%{_sysconfdir}/yum.repos.d/` will be better.